pipeline/sns.py

```diff
@@ -95,8 +91,18 @@ def get_topic_detail(topic: dict, detail: str) -> str | None:
         logging.error(f"Error getting topic details: {e}")
         return None
 
+def get_notification_distance(magnitude):
```

Comment:
I do not see a type hint or a doc string. Could you add these please.

Reply:
missed it sorry, fixed it now